templates/python/hgcBiasedGenProcesses_cfi.py

```diff
@@ -0,0 +1,150 @@
+                                                                                                      'SoftQCD:singleDiffractive = on',
+                                                                                                      'SoftQCD:doubleDiffractive = on'),
+                                                                      parameterSets = cms.vstring('pythia8CommonSettings',
+                                                                                                  'pythia8CUEP8M1Settings',
+                                                                                                  'processParameters')
+                                                                  )
+                                     )
+
+    #H->gg
+    if proc=='hgg':
+        process.generator = cms.EDFilter("Pythia8GeneratorFilter",
+                                         comEnergy = cms.double(14000.0),
+                                         crossSection = cms.untracked.double(8.45E-6),
+                                         filterEfficiency = cms.untracked.double(1.0),
+                                         maxEventsToPrint = cms.untracked.int32(1),
+                                         pythiaPylistVerbosity = cms.untracked.int32(1),
+                                         pythiaHepMCVerbosity = cms.untracked.bool(False),
+                                         PythiaParameters = cms.PSet( pythia8CommonSettingsBlock,
+                                                                      pythia8CUEP8M1SettingsBlock,
+                                                                      processParameters = cms.vstring('Main:timesAllowErrors    = 10000',
+                                                                                                      'HiggsSM:all=true',
+                                                                                                      '25:m0 = 125.0',
+                                                                                                      '25:onMode = off',
+                                                                                                      '25:onIfMatch = 22 22'       
+                                                                                                  ),
+                                                                      
+                                                                      parameterSets = cms.vstring('pythia8CommonSettings',
+                                                                                                  'pythia8CUEP8M1Settings',
+                                                                                                  'processParameters')
+                                                                  )
+                                    )
+    
+    #W->qq'
+    if proc=='wqq':
+        process.generator = cms.EDFilter("Pythia6GeneratorFilter",
```

Comment:
Is there a particular reason why W->qq needs to be Pythia6 while all others are Pythia8?